prototype/toUrl/toUrl.js

```diff
@@ -13,14 +13,24 @@
  * DO NOT modify the module.exports (for testing).
  * DO NOT declare any functions OUTSIDE the body of the toUrl function
  *  itself.
- *
- * @param {Object} ctx - The context from the Sample Generator
- * @param {Array} aspects - Array of one or more aspects
- * @param {Array} subjects - Array of one or more subjects
- * @returns {String} the url
  */
-function toUrl(ctx, aspects, subjects) {
-  return '';
-}
 
-module.exports = toUrl;
+module.exports = {
+ /*
+  * @param {Object} ctx - The context from the Sample Generator
+  * @param {Array} aspects - Array of one or more aspects
+  * @param {Array} subjects - Array of one or more subjects
+  * @returns {String} the url
+  */
+ toUrl(ctx, aspects, subjects) {
+
+  },
+
+  /* Helpers (optional)
+   *
+   * Define helper functions here if you need to be able to test them directly.
+   */
+  helpers: {
+
```

Comment:
Can you add the concatArray function here too, commented out? Someone should be able to setup a valid SGT just by using the prototype if they don't want to use a specific example, and it needs to be clear what the format is to define the helpers.